apps/backend/src/users/users.controller.ts

```diff
@@ -27,6 +28,31 @@ export class UsersController {
     return await this.usersService.remove(id);
   }
 
+  @Patch(':id/')
+  @ApiOperation({ summary: 'Edit Gender and Weight' })
+  @ApiQuery({ name: 'gender', required: false, type: String, enum: [Gender.Male, Gender.Female] })
+  @ApiQuery({ name: 'weight', required: false, type: String })
+  async update(
+    @Param('id', ParseUUIDPipe) id: string,
+    @Query('gender') gender?: Gender,
+    @Query('weight') weightString?: string //
+  ) {
```

Comment:
Instead of query parameters, I think it would be better to use the request body for these values, similar to other update methods (e.g. in the drink controller). You can create a DTO for this (with just the two values) and you can use class-validator to validate the values and won't have to do it in the controller method.